ibm/mas_devops/common_vars/casebundles/v8-231031-amd64.yml

```diff
@@ -0,0 +1,87 @@
+common_svcs_version: 1.19.4                  # Operator version 3.23.4 (https://github.com/IBM/cloud-pak/tree/master/repo/case/ibm-cp-common-services)
+db2u_version: 5.1.4                          # Operator version 110508.0.2 (https://github.com/IBM/cloud-pak/tree/master/repo/case/ibm-db2uoperator)
+events_version: 4.6.1                        # Operator version 4.6.1 (https://github.com/IBM/cloud-pak/tree/master/repo/case/ibm-events-operator)
+uds_version: 2.0.11                          # Operator version 2.0.11
+sls_version: 3.8.1                           # Operator version 3.8.1
+tsm_version: 1.5.1                           # Operator version 1.5.1
+dd_version: 1.1.5                            # Operator version 1.1.5
+appconnect_version: 6.2.0                    # Operator version 6.2.0
+cp4d_platform_version: 2.9.0+20230524.165553 # Operator version 3.8.0
+wsl_version: 6.5.0                           # Operator version 6.5.0
+wml_version: 6.5.0                           # Operator version 3.5.0
+spark_version: 6.5.0                         # Operator version 3.5.0
+
+# Watson discovery and its dependencies
+# Match corresponding case version for default versions in the catalog source
+# -----------------------------------------------------------------------------
+wd_version: 5.5.0                            # Operator version 4.6.5
+model_train_version: 1.2.7                   # Operator version 1.1.9
+elasticsearch_version: 1.1.1541              # Operator version 1.1.1541
+couchdb_version: 1.0.13                      # Operator version 2.2.1
+
+
+# Maximo Application Suite
+# -----------------------------------------------------------------------------
+mas_core_version:
+  8.10.x: 8.10.6  # Updated
+  8.11.x: 8.11.2  # Updated
+  8.9.x: 8.9.10   # No Update
+mas_assist_version:
+  8.10.x: 8.7.2   # No Update
+  8.11.x: 8.8.1   # Updated
+  8.9.x: 8.6.5    # No Update
+mas_hputilities_version:
+  8.10.x: 8.6.2   # No Update
+  8.11.x: ""      # Not Supported
+  8.9.x: 8.5.3    # No Update
+mas_iot_version:
+  8.10.x: 8.7.5   # Updated
+  8.11.x: 8.8.1   # Updated
+  8.9.x: 8.6.9    # No Update
+mas_manage_version:
+  8.10.x: 8.6.6   # Updated
+  8.11.x: 8.7.1   # Updated
+  8.9.x: 8.5.9    # No Update
+mas_monitor_version:
+  8.10.x: 8.10.5  # No Update
+  8.11.x: 8.11.1  # Updated
+  8.9.x: 8.9.6    # No Update
+mas_optimizer_version:
+  8.10.x: 8.4.1   # No Update
+  8.11.x: 8.5.0   # No Update
+  8.9.x: 8.3.3    # No Update
+mas_predict_version:
+  8.10.x: 8.8.2   # No Update
+  8.11.x: 8.9.0   # No Update
+  8.9.x: 8.7.2    # No Update
+mas_visualinspection_version:
+  8.10.x: 8.8.1   # No Update
+  8.11.x: 8.9.0   # No Update
+  8.9.x: 8.7.1    # No Update
+
+# Extra Images for UDS
+# ------------------------------------------------------------------------------
+uds_extras_version: 1.4.0
+
+# Extra Images for Mongo
+# ------------------------------------------------------------------------------
+mongo_extras_version: 4.4.21
```

Comment:
Can we have something as this going forward?

```
mongo_extras_version_default: 4.4.21
mongo_extras_version: "{{ lookup('env', 'MONGODB_VERSION') | default(mongo_extras_version_default, True) }}"
```

We need to have enough flexibility to set `MONGODB_VERSION` in our fvt pipelines so that we can have mongodb version being rotated in our daily fvt runs.

Reply:
Do you want me to make that change in this PR?? Don't want to mix that into catalog support PR. We can create a new PR for that once this is out. what do you think?